app/build.gradle

```diff
@@ -42,9 +42,9 @@ dependencies {
     implementation 'com.android.support:design:27.1.1'
 
     implementation 'com.jakewharton.timber:timber:4.7.1'
+    implementation 'com.squareup.tape2:tape:2.0.0-beta1'
```

Comment:
Can we not use generic methods to save files rather than using an external library?